pkg/cmd/attestation/inspect/inspect.go

```diff
@@ -93,96 +88,260 @@ func NewInspectCmd(f *cmdutil.Factory, runF func(*Options) error) *cobra.Command
+		_, err := opts.SigstoreVerifier.Verify([]*api.Attestation{a}, unsafeSigstorePolicy)
 
-	// If the user provides the --format=json flag, print the results in JSON format
-	if opts.exporter != nil {
-		details, err := getAttestationDetails(opts.Tenant, results)
+		// TODO: if the err is present, we keep on going because we want to be able to
+		// inspect bundles we might not have trusted materials for. but maybe we should
+		// print the error?
+		if err == nil {
+			inspectedBundle.Authentic = true
+		}
+
+		entity := a.Bundle
+		verificationContent, err := entity.VerificationContent()
 		if err != nil {
-			return fmt.Errorf("failed to get attestation detail: %v", err)
+			return fmt.Errorf("failed to fetch verification content: %w", err)
 		}
 
-		// print the results to the terminal as an array of JSON objects
-		if err = opts.exporter.Write(opts.Logger.IO, details); err != nil {
+		// summarize cert if present
+		if leafCert := verificationContent.GetCertificate(); leafCert != nil {
+
+			certSummary, err := certificate.SummarizeCertificate(leafCert)
+			if err != nil {
+				return fmt.Errorf("failed to summarize certificate: %w", err)
+			}
+
+			inspectedBundle.Certificate = CertificateInspection{
+				Summary:   certSummary,
+				NotBefore: leafCert.NotBefore,
+				NotAfter:  leafCert.NotAfter,
+			}
+
+		}
+
+		// parse the sig content and pop the statement
+		sigContent, err := entity.SignatureContent()
+		if err != nil {
+			return fmt.Errorf("failed to fetch signature content: %w", err)
+		}
+
+		if envelope := sigContent.EnvelopeContent(); envelope != nil {
+			stmt, err := envelope.Statement()
+			if err != nil {
+				return fmt.Errorf("failed to fetch envelope statement: %w", err)
+			}
+
+			inspectedBundle.Statement = stmt
+		}
+
+		// fetch the observer timestamps
+		tlogTimestamps, err := dumpTlogs(entity)
+		if err != nil {
+			return fmt.Errorf("failed to dump tlog: %w", err)
+		}
+		inspectedBundle.TransparencyLogEntries = tlogTimestamps
+
+		signedTimestamps, err := dumpSignedTimestamps(entity)
+		if err != nil {
+			return fmt.Errorf("failed to dump tsa: %w", err)
+		}
+		inspectedBundle.SignedTimestamps = signedTimestamps
+
+		inspectedBundles = append(inspectedBundles, inspectedBundle)
+	}
+
+	inspectionResult := BundleInspectResult{InspectedBundles: inspectedBundles}
+
+	// If the user provides the --format=json flag, print the results in JSON format
+	if opts.exporter != nil {
+		if err = opts.exporter.Write(opts.Logger.IO, inspectionResult); err != nil {
 			return fmt.Errorf("failed to write JSON output")
 		}
 		return nil
 	}
 
-	// otherwise, print results in a table
-	details, err := getDetailsAsSlice(opts.Tenant, results)
+	printInspectionSummary(opts.Logger, inspectionResult.InspectedBundles)
+
+	return nil
+}
+
+var heroHeader = `
+   _                       __ 
+  (_)__  ___ ___  ___ ____/ /_
+ / / _ \(_-</ _ \/ -_) __/ __/
+/_/_//_/___/ .__/\__/\__/\__/ 
+          /_/                 
+`
```

Comment:
Though the GitHub CLI hasn't published its accessibility guidance yet, ASCII banners like this are generally considered inaccessible to screen readers.